packages/components/Checkbox/src/Checkbox.styling.ts

```diff
@@ -54,6 +56,10 @@ export const stylingSettings: UseStylingOptions<CheckboxProps, CheckboxSlotProps
           alignItems: 'center',
           justifyContent: 'center',
         },
+        ...(Platform.OS === 'android' && {
+          pressRetentionOffset: typeof tokens.padding === 'number' ? tokens.padding : parseFloat(tokens.padding), /// Retention of the press area outside of the checkbox equal to padding to match accessibility requirement
+        }),
+        android_ripple: { color: tokens.rippleColor, radius: tokens.checkmarkSize, foreground: true },
       }),
       ['checkboxBackgroundColor', 'checkboxBorderColor', 'checkboxBorderRadius', 'checkboxBorderWidth', 'checkboxSize'],
```

Comment:
rippleColor should be added to the cache key

Reply:
added.

Reply:
Changes completed and approved , hence resolving.